beta/src/pages/learn/conditional-rendering.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: Conditional Rendering
+title: कंडिशनल  रेण्डरिंग
 ---
 
 <Intro>
 
-Your components will often need to display different things depending on different conditions. In React, you can conditionally render JSX using JavaScript syntax like `if` statements, `&&`, and `? :` operators.
+आपके कौम्पोनॅन्ट को अक्सर अलग-अलग स्थितियों के आधार पर अलग-अलग चीजें प्रदर्शित करने की आवश्यकता होगी। React में, आप सशर्त रूप से JSX को जावास्क्रिप्ट सिंटैक्स जैसे `if` स्टेटमेंट्स, `&&`, और `? :` ऑपरेटरों।
```

Comment:
`प्रदर्शित` => `डिस्प्ले`

I have been suggesting everyone to use this. As `प्रदर्शित` sounds a bit complex for the context

Reply:
👍